src/microbe_stage/editor/PatchMapNode.cs

```diff
@@ -27,6 +27,12 @@ public class PatchMapNode : MarginContainer
     [Export]
     public NodePath AdjacentPanelPath = null!;
 
+    /// <summary>
+    ///   For patches adjacent to a discovered patch that have not been entered by the player
```

Comment:
I think this comment needs to be worded to be a bit more vague, just so that we can have the more relaxed fog of war mode work and not contradict what the code comments say elsewhere.